recipes-support/ostree-pending-reboot/ostree-pending-reboot_0.1.bb

```diff
@@ -0,0 +1,39 @@
+SUMMARY = "OStree Pending Reboot service"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch systemd
+
+SRC_URI = " \
+	file://ostree-pending-reboot.service \
+	file://ostree-pending-reboot.timer.in \
+"
+
+S = "${WORKDIR}"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+# Value is in minutes (default to check for reboot every 1 minute)
+OSTREE_PENDING_REBOOT_CHECK_MINUTES ?= "1"
+
+PACKAGES = "${PN} ${PN}-timer"
```

Comment:
PN is already set into PACKAGES by default, so better use 'PACKAGES += "${PN}-timer"' instead.